movie-library/app/models/user.rb

```diff
@@ -0,0 +1,40 @@
+class User < ApplicationRecord
+    has_one :movie
+  
+    has_many :user_movie
+    has_many :movies, through: :user_movie
+  
+    # Validations
+    validates :name, presence: true
+    validates :ci, length: { in: 8..9 }, presence: true, allow_nil: false
+    validate :valid_ci
+  
+    def valid_ci
+      if ci
+        if ci.length < 8 || ci.length > 9
+          puts 'wrong length'
+          return false
+        end
+        validationNumber = ci[-1, 1].to_s
```

Comment:
This is and incorrect format for a backend variable. All variable should be snake case